starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -33,5 +34,10 @@ public InventoryController(InventoryDAO inventoryDAO) {
   public List<Inventory> findInventories() {
     return this.inventoryDAO.findAll();
   }
+
+  @PostMapping
+  public Inventory createInventory(@PostBody Inventory inventory) {
```

Comment:
The "@" symbol denotes a Java annotation or special data added to most code (e.g., methods, classes, variables).
`@PostBody` does not map `inventory` to the `@PostMapping`'s request body; a different annotation and its import would be needed.
View the `ProductController.java`'s `saveProduct` for using rest controller annotations.